MiniAn/minian_main.py

```diff
@@ -431,7 +432,112 @@ def load_doric_to_xarray(
 
     return varr, file_
 
-
+            
+def cross_register(minian_parameters, currentFile_AC, currentFile_A):
+    performCrossReg = minian_parameters.params_crossRegister["crossReg"]
+    if performCrossReg:
+        param_dist = 5 # Defines the max dist between centroids on different sessions to consider them as same cell
+        refFileName = minian_parameters.params_crossRegister["fname_crossReg"]
+        refImages   = minian_parameters.params_crossRegister["h5path_images"]
+        refRois     = minian_parameters.params_crossRegister["h5path_roi"]
+
+        # Load base/reference file to xarray
+        refCR, fileCR_ =  mn_utils.load_doric_to_xarray(refFileName, refImages, np.float64, None, "subset", None)
+
+        # max project of the miniAn images for base file and current file
+        ref_max     = refCR.max("frame")
+        current_max = currentFile_AC.max("frame")
+
+        result = xr.concat([ref_max, current_max], pd.Index(['session1', 'session2'], name = "session"))
```

Comment:
Can you change to double quotes `" "` for string and just single quotes `' '` for character (With exceptions like when you want have double quotes in a text like `'Bob say "Hello"'`).

I changed it every where just to make it the same as in (C/ C++) and consistent because some where `" "` and other `' '`.